SPEC/ovirt-openvswitch.spec

```diff
@@ -51,18 +60,18 @@ Summary:        Wrapper for openvswitch-ipsec rpm
 License:        Public Domain
 Requires:       openvswitch%{ovs_version}-ipsec
 Provides:       openvswitch-ipsec = %{ovs_version}
-Obsoletes:      openvswitch%{version_to_replace}-ipsec
+Obsoletes:      openvswitch%{ovs_to_replace}-ipsec
 
 %description ipsec
 Wrapper rpm for the base openvswitch-ipsec package
 
 %package        ovn
 Summary:        Wrapper for ovn rpm
 License:        Public Domain
-Requires:       ovn-%{ovn_version}
+Requires:       ovn%{?ovn_dash}%{ovn_version}
```

Comment:
I don't see the `ovn_dash` definition I guess it should be included only on el9 right?

Reply:
yes. note the "?"

Reply:
Noted, but shouldn't it be defined for el9? That's my original question/

Reply:
ah, yeah. it was in my commit but i see martin replaced/removed that. sorry i didn't notice.

Reply:
@mwperina please fix that. we need ovn-XX for the old one and ovnYY for the new one, hence the optional ovn_dash

Reply:
Hmm, I specifically search for `ovn_dash`  when merging commits and haven't found anything, no idea why.
Sorry, posted a fix